lib/open_api_spex/plug/cache.ex

```diff
@@ -1,50 +1,113 @@
 defmodule OpenApiSpex.Plug.Cache do
-  @moduledoc false
+  @moduledoc """
+  Cache for OpenApiSpex
 
-  @callback get(module) :: {OpenApiSpex.OpenApi.t(), map} | nil
-  @callback put(module, {OpenApiSpex.OpenApi.t(), map}) :: :ok
+  Settings:
 
-  @spec adapter() :: OpenApiSpex.Plug.AppEnvCache | OpenApiSpex.Plug.PersistentTermCache
-  def adapter do
-    if function_exported?(:persistent_term, :info, 0) do
+  ```elixir
+  config :open_api_spex, :cache_adapter Module
+  ```
+
+  There are already had three cache adapter:
+  * `OpenApiSpex.Plug.PersistentTermCache` - default
+  * `OpenApiSpex.Plug.AppEnvCache` - if VM not supported `persistent_term`
```

Comment:
I wonder if we should drop the `AppEnvCache` ? `persistent_term` was introduced in OTP 21.2, the current latest version is 23.0.

Reply:
If we do remove the `AppEnvCache`, then it would simplify the configuration for library users.
They could set a boolean in their config like:

```elixir
# dev.exs
config, :open_api_spex, :cache, false
```

Reply:
Now we setting this lib supported elixir version in `mix.exs`: `elixir: "~> 1.7"`
If we just use `persistent_term` will let older elixir version can't use

Reply:
Your can visit [Compatibility between Elixir and Erlang/OTP](https://hexdocs.pm/elixir/1.10.4/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp)  to see how `elixir` `1.7` is supported for `OTP` `19 - 22`
If just use `persistent_term` we should set upgrade to `1.10` from `1.7` in `mix.exs` like this:
```elixir
elixir: "~> 1.10"
```

Reply:
Fair point. Elixir has only _required_ OTP 21+ since 1.10.
Let's keep the `AppEnvCache` for a bit longer 👍